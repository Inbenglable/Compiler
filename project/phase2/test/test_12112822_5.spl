int foo(int i){
    int i = 1;
    return i;
}
int main(){
    foo(1);
    return 0;
}