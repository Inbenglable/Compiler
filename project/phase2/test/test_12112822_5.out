Error type 3 at line 2: variable "i" is redefined in the same scope
